ts/features/bonus/bonusVacanze/store/reducers/index.ts

```diff
@@ -36,7 +38,8 @@ const bonusReducer = combineReducers<BonusState, Action>({
   bonusVacanze: bonusVacanzeReducer,
   bpd: bpdReducer,
   cgn: cgnReducer,
-  sv: svReducer
+  sv: svReducer,
+  cdc: CdcReducer
```

Comment:
This is _really_ minor, but maybe the `CdcReducer` should be written in camel case (E.g. `cdcReducer`)? 🤓

Reply:
Sure, thank you! Fixed in [70335d1](https://github.com/pagopa/io-app/pull/3907/commits/70335d179f541d964c3bdcd4451943bd3f41bff3)